test/e2e/testdata/pipelinerungcs.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: hello-gcs
```

Comment:
I had to re-add this file as `hello-gcs` because `hello`pr causes indeterministic behaviour while testing:
```
    e2e_gcs_test.go:56: Error creating PipelineRun: object is being deleted: pipelineruns.tekton.dev "hello" already exists
```